tests/func/test_live.py

```diff
@@ -66,6 +66,7 @@ def live_stage(tmp_dir, scm, dvc, mocker):
         pytest.skip("no dvclive")
 
     mocker.patch("dvc.stage.run.Monitor.AWAIT", 0.01)
+    mocker.patch("dvc.repo.live.webbrowser_open")
```

Comment:
I'm worried that this may get unnoticed on new tests. We should mock `webbrowser.open` globally if that's possible.